mitodl/mitxonline
# Jkachel/466 display refund tags on receipt

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #466 

#### What's this PR do?
Fixes #466. Adds a label and readout for refunds applied to an order.

#### How should this be manually tested?
_This would be better tested after #480 is merged. You can manually add Transactions to test, though._

From the Order History page, select an order that has been refunded. The refund(s) should show in the order summary on the right. If the refund has multiple refunds, each will be listed. (The system doesn't currently have UI support for multiple partial refunds but it can be done by either manually adding Transaction records in or by flipping an order to Fulfilled after it has been set to Refunded, and then refundng it again.) 

#### Notes
- Wasn't real sure if the total displayed should be the order total or the actual amount paid (which, after a refund, would be the order total minus refunds), so it doesn't adjust the total now. (In the screenshots below, the effective cost for that order is zero since the entire order was refunded.) 
- The refund date is available in the refund store, but not displayed - do we want to display this? (The reason is explicitly not  transmitted as that seems like a bad idea as there may be order reasons that are not to be viewed by the learner.) 

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/945611/159080979-d79ddc65-335c-471a-ad52-8a2686601157.png)
![image](https://user-images.githubusercontent.com/945611/159081030-2e584a91-7a95-479d-9e68-6e2b8929cb66.png)
